content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,337 @@
+.. note::
+   Some fields only appear if specific settings and options have been configured.
+
+Sales section
+~~~~~~~~~~~~~
+
+In the :guilabel:`Sales` section of the :guilabel:`Other Info` tab, there are sales specific fields
+that can be configured.
+
+.. image:: create_quotations/other-info-sales.png
+   :align: center
+   :alt: The Sales section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Salesperson`: assign a salesperson from the drop-down menu to be associated with this
+  quotation. The user who originally created the quotation is selected in this field, by default.
+- :guilabel:`Sales Team`: assign a specific sales team to this quotation. If the selected
+  :guilabel:`Salesperson` is a member of a sales team, that team is auto-populated in the field.
+- :guilabel:`Company`: select a company from the drop-down menu that this quotation should be
+  associated with. This field only appears when working in a multi-company environment.
+- :guilabel:`Online signature`: tick this checkbox to request an online signature from the customer
+  to confirm the order. This field only appears if the *Online Signature* setting has been enabled.
+- :guilabel:`Online payment`: tick this checkbox, and enter a desired percentage in the adjacent
+  field, in order to request an online payment from the customer (for that designated percentage of
+  the total amount) to confirm the order. This field only appears if the *Online Payment* setting
+  has been enabled.
+- :guilabel:`Customer Reference`: choose to enter a custom reference ID for this customer. The
+  entered reference ID can contain letters, numbers, or a mix of both.
+- :guilabel:`Tags`: choose to add specific tags to the quotation for added organization and enhanced
+  searchability in the Odoo *Sales* application. Multiple tags can be added, if necessary.
+
+Delivery section
+~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Delivery` section of the :guilabel:`Other Info` tab, there are delivery-specific
+fields that can be configured.
+
+.. image:: create_quotations/other-info-delivery.png
+   :align: center
+   :alt: The Delivery section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Shipping Weight`: the weight of the items being shipped. This field is not modifiable.
+  Product weight is configured on individual product forms.
+- :guilabel:`Incoterm`: select an Incoterm (or International Commerical Term) to use as predefined
+  commerical terms for international transactions.
+- :guilabel:`Incoterm Location`: if an Incoterm is being used, enter in the international location
+  in this field.
+- :guilabel:`Shipping Policy`: select a desired shipping policy from the drop-down menu. If all
+  products are delivered at once, the delivery order is scheduled based on the greatest product lead
+  time. Otherwise, it is based on the shortest lead time. The available options are:
+  :guilabel:`As soon as possible` or :guilabel:`When all products are ready`.
```

Comment:
early line break
```suggestion
  time. Otherwise, it is based on the shortest lead time. The available options are: :guilabel:`As
  soon as possible` or :guilabel:`When all products are ready`.
```